README.md

```diff
@@ -1,4 +1,7 @@
 ![](images/coverage.svg)
+[![Python version](https://img.shields.io/badge/Python-3.8-blue?style=plastic)](https://www.python.org/)
+[![GitHub Actions Status](https://github.com/LSARP/lrg-omics/actions/workflows/python-package.yml/badge.svg?branch=develop)](https://github.com/LSARP/lrg-omics/actions/?query=workflow)
```

Comment:
IMO, and to keep in mind as a to-do item in the near future, it would be nice to change the name of the GitHub actions file from `python_package.yml` to `CI.yml`. It would make the name on the badge self-explanatory.